config/locales/links/en.yml

```diff
@@ -12,6 +12,7 @@ en:
     contact_support: Contact %{app_name} Support
     continue_sign_in: Continue sign in
     create_account: Create an account
+    exit_login: 'Exit Login.gov'
```

Comment:
(nit) Can you replace "Login.gov" with `%{app_name}` in these translations?

Reply:
This was explicit in the design, so let me check with UX real quick

Reply:
There is already a translation for Exit here: https://github.com/18F/identity-idp/blob/main/config/locales/idv/en.yml#L19

Reply:
(And now that I see the failing tests I'm afraid I misunderstood your comment @matthinz. Yes, i18n linter says we need to use app_name)